plugins/inputs/ethtool/ethtool_linux.go

```diff
@@ -256,7 +256,7 @@ func (c *CommandEthtool) Interfaces(includeNamespaces bool) ([]NamespacedInterfa
 
 			handle, err := netns.GetFromPath(filepath.Join(namespaceDirectory, name))
 			if err != nil {
-				c.Log.Warnf(`Could not get handle for namespace "%s": %s`, name, err)
+				c.Log.Warnf("Could not get handle for namespace %q: %w", name, err)
```

Comment:
`%w` should be only used for creating error, not for printing. Use `%s` or `%v` instead.